curriculum/challenges/english/25-front-end-development/lecture-working-with-objects/67329f737126b75bcb949e13.md

```diff
@@ -12,7 +12,7 @@ Watch the video or read the transcript and answer the questions below.
 
 # --transcript--
 
-What is a JavaScript object, and how can you access properties from an object?
+What is an object in JavaScript, and how can you access properties from an object?
```

Comment:
Technically, this is what is said in the video though. So I am not sure this change is needed. Transcripts should be the written document of what is said in the video. (With exception of the explanation for visuals)

Ideally, the title and what was said would match exactly, but since Beau said it a little bit differently, we can just keep this as is IMO

Reply:
considering this comment I am closing the PR